editor/src/messages/tool/common_functionality/shape_editor.rs

```diff
@@ -148,6 +152,36 @@ impl ShapeState {
 		self.selected_shape_state.values().flat_map(|state| &state.selected_points)
 	}
 
+	/// Moves an anchor to a new position, new_position must be in art board space.
+	/// Returns Some(()) if successful and None otherwise.
+	pub fn reposition_anchor_point(&self, point: &ManipulatorPointId, responses: &mut VecDeque<Message>, document: &Document, new_position: DVec2, layer_path: &[u64]) -> Option<()> {
+		if point.manipulator_type.is_handle() {
+			trace!("cannot call `reposition anchor` on a handle");
```

Comment:
This is probably better as a warning `warn!()`.

Reply:
agreed and fixed.